Core/Frameworks/Baikal/Resources/Templates/Main.html

```diff
@@ -0,0 +1,14 @@
+{% autoescape false %}
+<header class="jumbotron subhead" id="overview" style="padding-bottom:5em;">
+	<h1><img style="height:150px; float:left;" src="res/core/Baikal/Images/logo-baikal.png" /> Baïkal
+	<p class="lead">Lightweight CalDAV+CardDAV server.</p></h1>
+</header>
+
+{{ message }}
+
+<section id="about">
+	<div class="page-header">
+		<h1>Baïkal is running alright.</h1>
+	</div>
+</section>
+{% endautoescape %}
```

Comment:
Please add a line break

Reply:
fixed in 9dac98d6d4c42e3c1ab9c15ab2e93ab1e0bc87be